ivc/src/ivc/interpreter.rs

```diff
@@ -913,15 +916,101 @@ where
 {
 }
 
+/// Builds selectors for the IVC circuit.
+pub fn build_selectors<F, const N_COL_TOTAL: usize, const N_CHALS: usize>(
+    domain_size: usize,
+) -> Vec<Vec<F>>
+where
+    F: PrimeField,
+{
+    // 3*N + 6*N+2 + N+1 + 35*N + 5 + 1 + N_CHALS =
```

Comment:
Maybe change the order to `N_CHALS + 1` so that it matches the order of the for-loops below.

Reply:
Done